geomesa-cassandra/geomesa-cassandra-tools/pom.xml

```diff
@@ -76,6 +65,13 @@
             <groupId>org.geotools</groupId>
             <artifactId>gt-epsg-wkt</artifactId>
         </dependency>
+
+        <!-- provided dependencies -->
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-compiler</artifactId>
+            <scope>provided</scope>
```

Comment:
Was this compile before?  Is this just clean up?

Reply:
it was provided before, some of the poms were just incorrectly commented/sectioned on provided deps vs compile deps, so i moved them around a little to group them together correctly.
the one actual change is that i did change the scope of zookeeper from 'compile' to 'provided' in all the 'geoesa-*-tools' modules - it didn't actually change the install tgzs produced, and i'm not sure why it was compile scope as it should have been provided all along...